lib/src/localization.dart

```diff
@@ -47,6 +51,12 @@ class Localization {
     return res;
   }
 
+  String _replaceNamedArgs(String res, Map<String, String> args) {
+    if(args == null || args.isEmpty) return res;
+    args.forEach((String key, String value) => res = res.replaceAll(RegExp('{{$key}}'), value));
```

Comment:
@3ace whu you use double brackets `{{$key}} `? I think beter use sigle bracket `{$key}`

Reply:
yep, nice look

Reply:
@Overman775 I think double curly braces could differentiate the key with any other regular string, just in case there's user that put some string inside curly braces.

But if you think single curly braces is more preferable, I'll update my code.

Reply:
@3ace single curly braces more preferable, single style easier to understand

Reply:
Okay understood, I'll update the code